src/pkg/export/export.go

```diff
@@ -16,28 +22,50 @@ type Collection interface {
 	Items(context.Context) <-chan Item
 }
 
-// ItemData is the data for an individual item.
-type ItemData struct {
-	// Name is the name of the item. This is the name that the item
-	// would have had in the service.
-	Name string
+// BaseCollection holds the foundational details of an export collection.
+type BaseCollection struct {
+	// BaseDir contains the destination path of the collection.
+	BaseDir string
 
-	// Body is the body of the item. This is an io.ReadCloser and the
-	// SDK consumer is responsible for closing it.
-	Body io.ReadCloser
+	// BackingCollection is the restore collection from which we will
+	// create the export collection.
+	BackingCollection data.RestoreCollection
```

Comment:
`BackingCollection` sometimes have to be `[]data.RestoreCollection` for cases like pst export.

Reply:
Good to know!  I'll get that fixed up.